test/e2e/framework/unmanaged.go

```diff
@@ -213,8 +212,8 @@ func (f *UnmanagedFramework) ClusterDynamicClients(apiResource *metav1.APIResour
 		// Check if this cluster is the same name as the host cluster name to
 		// make it the primary cluster.
 		testClusters[clusterName] = common.TestCluster{
-			clusterConfig,
-			client,
+			TestClusterConfig: clusterConfig,
```

Comment:
What rule is this violating exactly?

Reply:
Use the style above to initialise structs would reduce the possibility of unexpected initialises error, suppose we have something like:

```go
type Test struct {
  a int
  b int
}
```

Initialise the struct above use `Test{1, 2}` may lead to problems when the order of vars changes in the `Test` struct.

And there are only two places which use this approach to initialise the struct. It's better to maintain consistency in the project.

Reply:
I get the point of the check, but I don't think it's worth enforcing at this time.  Please disable it for now.